plugins/esi/fetcher/HttpDataFetcherImpl.cc

```diff
@@ -80,7 +80,7 @@ HttpDataFetcherImpl::addFetchRequest(const string &url, FetchedDataProcessor *ca
   if (length < (int)sizeof(buff)) {
     http_req = buff;
   } else {
-    http_req = (char *)malloc(length + 1);
+    http_req = static_cast<char *>(malloc(length + 1));
```

Comment:
I'm not convinced by ``static_cast`` on ``malloc()``. It does nothing but look uglier. It would be better not to include so many different changes in this PR.